test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -303,10 +305,9 @@ instance Arbitrary PoolId where
     arbitrary = PoolId . T.pack <$> replicateM 3 arbitraryPrintableChar
 
 instance Arbitrary WalletId where
-    arbitrary = WalletId . uuidFromWords <$> arbitrary
-
-uuidFromWords :: (Word32, Word32, Word32, Word32) -> UUID.UUID
-uuidFromWords (a, b, c, d) = UUID.fromWords a b c d
+    arbitrary = do
+        bytes <- BS.pack <$> replicateM 16 arbitrary
```

Comment:
why 16 here, because address pool gap is 16?

Reply:
No. Just arbitrary.